Dockerfile

```diff
@@ -89,7 +89,7 @@ ARG HLS_VERSION=1.7.0.0
 RUN \
   set -o errexit -o xtrace; \
   if test -n "$HLS_VERSION"; then \
-  ghcup install hls "$HLS_VERSION" --set; \
+  ghcup compile hls -g 7760340e --ghc $GHC_VERSION -j4 --cabal-update; \
```

Comment:
I have some random thoughts about this:

- Can we use long flags for everything? `-g` is `--git-ref` and `-j` is `--jobs`. 
- Similarly, can we use the complete Git commit hash of `7760340e999693d07fdbea49c9e20a3dd5458ad3`?
- Although I don't expect it to be necessary, can we be sure to quote the `GHC_VERSION` variable? 
- Should we use `HLS_VERSION` as the Git ref? That way `1.7.0.0` would work by targeting a tag and `7760340e` would work by targeting a commit. 
- How long will this take? And does it make sense to set `-j4` rather than `-j`? The build machine could have any number of cores available.